collect_app/src/main/java/org/odk/collect/android/injection/config/AppDependencyModule.java

```diff
@@ -584,7 +587,17 @@ public UpgradeInitializer providesUpgradeInitializer(Context context, SettingsPr
 
     @Provides
     public ApplicationInitializer providesApplicationInitializer(Application context, UserAgentProvider userAgentProvider, PropertyManager propertyManager, Analytics analytics, UpgradeInitializer upgradeInitializer, AnalyticsInitializer analyticsInitializer, ProjectsRepository projectsRepository, SettingsProvider settingsProvider) {
-        return new ApplicationInitializer(context, userAgentProvider, propertyManager, analytics, upgradeInitializer, analyticsInitializer, projectsRepository, settingsProvider);
+        return new ApplicationInitializer(context, userAgentProvider, propertyManager, analytics, asList(
```

Comment:
We might want to introduce dependencies for `Initializer`s in the future but at the moment it's nice to be able to see the order they are executed in to make sure any interdependencies between them work out ok.